rainbowsaddle/__init__.py

```diff
@@ -164,6 +178,7 @@ def main():
             'rainbow-saddle PID')
     parser.add_argument('--gunicorn-pidfile', help='a filename to store the '
             'gunicorn PID')
+    parser.add_argument('--statsd', help='Set it to true to enable statsd metrics')
```

Comment:
Pretty sure you can specify `action='store_true'` here which should force statsd to be False if not set, otherwise True, which prevents the need for the string comparisons / case checks.